izug/refegovservice/profiles/default/types/izug.refegovservice.egovservice.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    name="izug.refegovservice.egovservice"
+    meta_type="Dexterity FTI"
+    i18n:domain="izug.refegovservice">
+
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">EGov Service</property>
+  <property
+      i18n:translate=""
+      name="description">Description of an Government Service</property>
+
+  <property name="allow_discussion">False</property>
+  <property name="factory">izug.refegovservice.egovservice</property>
+  <property name="icon_expr">string:${portal_url}/document_icon.png</property>
```

Comment:
```suggestion
  <property name="icon_expr"></property>
```

We use the icons defined in SCSS
- https://github.com/4teamwork/ftw.theming#icons
- https://github.com/4teamwork/ftw.book/blob/5b3ad2b0b5a21ee029ee0c291ac6838e8e7bde74/ftw/book/browser/resources/theming.scss#L2